### DISCUSSION TITLE
I can't see dirs sidebar inside of an article.

### DISCUSSION THREAD

**Discussion by David7ce** [imfing/hextra#59] [Q&A] [answered]
Here's an example.

```
.
├── ciencia
│   ├── cientificos.md
│   ├── _index.md
│   ├── listado-experimentos.md
│   └── nota-ciencia.md
├── conceptos
│   ├── _index.md
│   ├── toroide.md
│   └── vacio.md
```

When I click on _index I see the left sidebar with directories and files. But inside the files the left sidebar disappears.

This is the yaml of an "ciencia/_index.md":
```
---
title: Ciencia
weight: 2
sidebar:
  open: true
---
```

This is the yaml of "ciencia/cientificos.md":
```
---
title: Científicos
---
```

**Comment by imfing** [comment, marked as answer]
See previous discussion: #18 

To enable the documentation layout with a sidebar, you'll need to specify `type: docs` in the frontmatter of your Markdown file. This is necessary because your page resides outside the standard `docs/` directory. 
The default layout for single pages (such as the `About` page) doesn't include a sidebar, which is why you're not seeing it on your page.

**Reply by David7ce** [reply]
Thanks for your quick response, I will try it.
